### PR TITLE
Mybranch

### PR DESCRIPTION
**Pull request by LailaL88**
Added the styles to top part of site and created a modal

### CODE REVIEW

File: wp-content/themes/poetiktheme/src/app.scss [hunk 1]
```diff
@@ -43,12 +44,17 @@ body {
...
-h2,
-h3,
-p {
+// body,
+// h1,
+// h2,
+// h3,
+// p {
+//     margin: 0;
+// }
```
Comment (srlhyo) [inline wp-content/themes/poetiktheme/src/app.scss:53]: Why did you comment out?

**Review by srlhyo — CHANGES_REQUESTED** [review]
please review my comments

File: wp-content/themes/poetiktheme/src/app.scss (hunk 1 above)
Comment (LailaL88) [reply wp-content/themes/poetiktheme/src/app.scss:53]: yes

**Comment by srlhyo** [comment]
closed

**Pull request closed without merging**